### DISCUSSION TITLE
Can I control smart light/devices with this

### DISCUSSION THREAD

**Discussion by Josh65-2201** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#364] [Q&A] [answered]
^

**Comment by avm99963** [comment, marked as answer]
Yes you can! If I'm not mistaken, when you try to do this, a prompt will appear in the Google Home app to allow personal results.

**Reply by Josh65-2201** [reply]
The google settings crashes when toggling it on then doesn't save it for me. have you got it to work before?

**Reply by avm99963** [reply]
Woops, that's true: I'm also experiencing the same while setting up personal results (the Google Home app crashes) :/

I just saw there's an issue open: #175.

**Reply by avm99963** [reply]
However, I can control devices even if I'm not able to allow personal results because of that bug.

**Reply by Harsharma2308** [reply]
Same i can control smart lights even though personal results don't work. Smh the smart lights thing is out of the personal results' feature for me.

**Reply by MadmaxHart** [reply]
This is great. I went to this GitHub so I can control my smart lights above my desk from my computer and this works great. It only took me about an hour to set up since I kept getting an error when authenticating (very well could have been me). But this works great. Thank you so much. My only suggestion is I would like to be able to set this to a room (e.g. Bedroom, Living Room, Computer Room, etc.) (I will make a new request if one is not already made).

**Reply by CptPlanet89** [reply]
Samesies. After much troubleshooting, I got it goin, more or less, but she's still too hesitant to open my "personal" calendar but has no problem turning on the "Sex Lamps' in the bedroom. Not complaining though, much obliged to have this much! 

Follow up, any direct control over the apps on the PC? Can't seem to get the good girl to open up Chrome for me..

**Reply by im-coder-lg** [reply]
GAUDC can't open apps by default, it is a limitation as of now.

**Comment by obed9057** [comment]
Actually you cant because this is not a google console smart home project in order to do this function contact the developer or make your own google home client at google developer console. Sent from Mail for Windows 10 From: Adrià Vilanova MartínezSent: May 10, 2021 6:48 AMTo: Melvin-Abraham/Google-Assistant-Unofficial-Desktop-ClientCc: SubscribedSubject: Re: [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client] Can I control smart light/devices with this (#364) Yes you can! If I'm not mistaken, when you try to do this, a prompt will appear in the Google Home app to allow personal results.—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.
